webapp/components/wpt-flags.js

```diff
@@ -101,8 +108,26 @@ wpt.FlagsClass = (superClass, readOnly, useLocalStorage) => class extends superC
   static get properties() {
     const props = {};
     makeFeatureProperties(props, wpt.ClientSideFeatures, readOnly, useLocalStorage);
+    if (useLocalStorage) {
```

Comment:
Do we still need to condition on localStorage?

Reply:
Without it, `WPTEnvironmentFlagsEditor` (I believe that's the page for server-side flags) will also try to define this method, which is not that useful because it's a client-side flag only and we don't have to manage it?